ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -159,15 +163,20 @@ private void SetSetting<T>(string key, T value)
             var values = ApplicationData.Current.LocalSettings.Values;
             values[key] = value;
 #else
-            var values = ConfigurationManager.AppSettings;
-            if (values[key] == null)
+            var configFile = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
```

Comment:
The ConfigurationManager.AppSettings itself is read-only so setting directly on the values collection causes a System.Configuration.ConfigurationErrorsException with "The configuration is read only."
The code I added was straight from MSDN... 
The code may or may not be what we ultimately want...